core/trino-main/src/main/java/io/trino/sql/rewrite/ShowQueriesRewrite.java

```diff
@@ -292,26 +293,28 @@ protected Node visitShowRoles(ShowRoles node, Void context)
         {
             Optional<String> catalog = node.getCatalog().map(c -> c.getValue().toLowerCase(ENGLISH));
 
+            List<Expression> rows;
```

Comment:
please extract method to obtain rows

Reply:
No. I will extract a method to handle rows. It will be better.